src/Compilers/Core/MSBuildTask/Microsoft.Managed.EditorConfig.targets

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
ToolsVersion=16.1?

Reply:
I don't think that does what we think it does. @rainersigwald When should I set the `ToolsVersion` attribute of a `<Project>` element, and what are the valid values?

Reply:
Never set it, and especially never set it in an imported file--it's only relevant to top-level projects and it's mostly irrelevant there.

The valid values depend on the build environment: VS 2019 supports `Current`, and `4.0` (and maybe some other old .NET Framework versions, but don't use any of those!).

Failing to specify it for a project means "use the one that's built into the MSBuild that's currently running", which is almost certainly what you want.

Reply:
Removed.